### DISCUSSION TITLE
Use Vite instead of Webpack

### DISCUSSION THREAD

**Discussion by sadeghbarati** [idiotWu/smooth-scrollbar#401] [Ideas]
**Vite** is growing so fast, documentation and API is easy to understand, also have the easy configuration

Unlike **Webpack** documentation and API are difficult to understand

Compiled js with **Webpack** have unnecessary code or unwanted polyfills

**Comment by dolphin-wood** [comment]
Not familiar with vite but I think we can give it a try 😆
